### PR TITLE
OTG Test: P4RT-7.1: LLDP: PacketIn

### LINKED ISSUE #1004: OTG: P4RT-7.1: LLDP: PacketIn

### PR DESCRIPTION
**Pull request by ANISH-GOTTAPU**
OTG conversion of P4RT-7.1: LLDP: PacketIn [here](https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/p4rt/ate_tests/lldp_packetin_test/lldp_packetin_test.go)

resolves #1004

### CODE REVIEW

File: feature/experimental/p4rt/otg_tests/lldp_packetin_test/lldp_packetin_test.go [hunk 1]
```diff
@@ -508,3 +515,12 @@ func (lldp *LLDPPacketIO) GetEgressPort() []string {
 func (lldp *LLDPPacketIO) GetIngressPort() string {
 	return lldp.IngressPort
 }
+
+func ateInterface(t *testing.T, topo gosnappi.Config, portID string) gosnappi.Port {
```
Comment (michaelore) [inline feature/experimental/p4rt/otg_tests/lldp_packetin_test/lldp_packetin_test.go:519]: Remove the t *testing.T parameter? It's unused.

Comment (ANISH-GOTTAPU) [reply feature/experimental/p4rt/otg_tests/lldp_packetin_test/lldp_packetin_test.go]: Removed the unused parameter

**Review by michaelore — APPROVED** [review]
Thanks!

**Pull request merged**